charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 10.2.0
+version: 10.3.1
```

Comment:
Why are you gong from 10.2.0 to 10.3.1?
should be 10.3.0 or not?

Reply:
whops, you are right! sorry about that.